From 08337d9a9dd3fede53f2814bcc2d7fc517b495f6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Johan=20Heikkil=C3=A4?= <johan.heikkila@gmail.com>
Date: Fri, 15 Jul 2016 21:40:09 +0300
Subject: [PATCH 1/6] Fedora ships with quazip


diff --git a/sleepyhead/UpdaterWindow.cpp b/sleepyhead/UpdaterWindow.cpp
index 90217bc..cae3405 100644
--- a/sleepyhead/UpdaterWindow.cpp
+++ b/sleepyhead/UpdaterWindow.cpp
@@ -23,8 +23,8 @@
 #include <QProcess>
 
 #include "SleepLib/profiles.h"
-#include <quazip/quazip.h>
-#include <quazip/quazipfile.h>
+#include <quazip5/quazip.h>
+#include <quazip5/quazipfile.h>
 #include "UpdaterWindow.h"
 #include "ui_UpdaterWindow.h"
 #include "version.h"
diff --git a/sleepyhead/sleepyhead.pro b/sleepyhead/sleepyhead.pro
index c72e842..f9371be 100644
--- a/sleepyhead/sleepyhead.pro
+++ b/sleepyhead/sleepyhead.pro
@@ -70,7 +70,7 @@ DEFINES += BETA_BUILD
 
 
 unix:!macx:!haiku {
-    LIBS            += -lX11 -lz -lGLU
+    LIBS            += -lX11 -lquazip5 -lz -lGLU
     DEFINES         += _TTY_POSIX_
 }
 
@@ -288,9 +288,10 @@ mac {
     QMAKE_BUNDLE_DATA += TransFiles
 }
 
-include(../3rdparty/quazip/quazip/quazip.pri)
-INCLUDEPATH += $$PWD/../3rdparty/quazip
-DEPENDPATH += $$PWD/../3rdparty/quazip
+# Fedora ships with quazip, not needed
+#include(../3rdparty/quazip/quazip/quazip.pri)
+#INCLUDEPATH += $$PWD/../3rdparty/quazip
+#DEPENDPATH += $$PWD/../3rdparty/quazip
 
 DISTFILES += \
     ../README
-- 
2.7.4

