From 40f27771499c1731ceb67741dd9b472050ada1eb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Johan=20Heikkil=C3=A4?= <johan.heikkila@gmail.com>
Date: Fri, 15 Jul 2016 21:59:17 +0300
Subject: [PATCH 4/6] Greek and Arabic translations not yet available


diff --git a/SleepyHeadQT.pro b/SleepyHeadQT.pro
index 932e458..b80ae6e 100644
--- a/SleepyHeadQT.pro
+++ b/SleepyHeadQT.pro
@@ -15,5 +15,3 @@ TRANSLATIONS += \
     Translations/Bulgarian.bg.ts \
     Translations/English.en_UK.ts \
     Translations/Portugues.pt.ts \
-    Translations/Arabic.ar.ts \
-    Translations/Greek.el.ts
-- 
2.7.4

